_posts/convars/2022-02-02-mom_ui_time_default_precision.md

```diff
@@ -0,0 +1,11 @@
+---
+title: mom_ui_time_default_precision
+category: var
+tags:
+  - ui
```

Comment:
Maybe add `hud` and `timer` tags just to put it in that category - I can imagine someone looking for changing precision by looking at the timer tag.